terraform/modules/ecs/locals.tf

```diff
@@ -8,6 +8,6 @@ locals {
   nih_cidr_ranges                 = ["129.43.0.0/16", "137.187.0.0/16", "10.128.0.0/9", "165.112.0.0/16", "156.40.0.0/16", "10.208.0.0/21", "128.231.0.0/16", "130.14.0.0/16", "157.98.0.0/16", "10.133.0.0/16"]
   vpc_cidr                        = data.aws_vpc.current.cidr_block
   os_domain_name                  = var.add_opensearch_permission ? "${var.resource_prefix}-opensearch" : "*"
-  permission_boundary_arn         = var.env != "stage" || var.env != "prod" ? "arn:aws:iam::${data.aws_caller_identity.current.account_id}:policy/PermissionBoundary_PowerUser" : null
+  permission_boundary_arn         = terraform.workspace == "dev" || terraform.workspace == "dev2" || terraform.workspace == "qa" || terraform.workspace == "qa2" ?  "arn:aws:iam::${data.aws_caller_identity.current.account_id}:policy/PermissionBoundary_PowerUser" : null
```

Comment:
can we make this skip the permission boundary if it is not stage or prod:

permission_boundary_arn = terraform.workspace == "prod" || terraform.workspace == "stage" ? "" : "arn:aws:iam::${data.aws_caller_identity.current.account_id}:policy/PermissionBoundary_PowerUser"